src/main/java/org/students/Transaction.java

```diff
@@ -0,0 +1,57 @@
+package org.students;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class Transaction implements Serializable {
+    String transactionHash; //- calculated SHA-256 hash of the transaction
+
+    List<TransactionOutput> inputs; // - UTXO’s used as inputs for given transaction
+
+    List<TransactionOutput> outputs; // - outputs produced by given transaction
+
+    String senderPublicKey; // - base58 encoded string representing sender’s public key
+
+    LocalDateTime timestamp; // - timestamp of when the transaction was made
+
+    public String getTransactionHash() {
```

Comment:
I advise using Lombok, which uses annotations to automatically generate getter/setter methods. Makes code slightly cleaner.